### ISSUE TITLE
Dynamic not working

### ISSUE THREAD

**Issue by Rush** [pheuter/require-j#4]
I would like to use your plugin but I am in need to run both a development-unbuilt version and production-build version side by side. It would be great to see the dynamic support improved. Currently the `demo-dynamic` yields simply `undefined` for the jade require.

**Comment by pheuter** [comment]
@RushPL Yes, this is a known feature that I am (slowly) working towards. Ran into a few snags with the latest version of jade as it isn't as browser-friendly anymore.

**Comment by Rush** [comment]
I made a new plugin that follows upstream Jade and has include, extend support
https://github.com/encharm/require-jade-ng

Please comment on encharm/require-jade-ng#1

**Comment by pheuter** [comment]
Oh nice! It's been a while since the last comment, heh. I'll take a look soon, do you have a test suite setup?

**Comment by Rush** [comment]
Not yet, and it's setup for development on Linux/Unix atm, with a simple Makefile. If you write `make test` it will create a built version in `test/`. Running a static server there and trying out `index.html` and `index-built.html` is the only test atm. I am planning on doing some phantom.js automatic test setup in the future.

**Comment by pheuter** [comment]
Sounds good
